playground/package.json

```diff
@@ -19,25 +19,24 @@
     "tsc:watch": "tsc -p ./ --noEmit -w"
   },
   "dependencies": {
-    "@babel/polyfill": "7.4.4",
     "@types/jest": "24.0.11",
     "@types/node": "11.13.8",
     "@types/prop-types": "15.7.1",
     "@types/react": "16.8.14",
     "@types/react-dom": "16.8.4",
     "@types/react-redux": "7.0.8",
     "@types/react-router-dom": "4.3.2",
-    "@types/react-router-redux": "5.0.18",
-    "axios": "0.18.0",
+    "axios": "^0.19.0",
+    "connected-react-router": "6.5.0",
     "cuid": "2.1.6",
     "prop-types": "15.7.2",
     "react": "16.8.6",
     "react-dom": "16.8.6",
-    "react-redux": "7.0.2",
+    "react-redux": "7.1.0",
     "react-redux-typescript-scripts": "1.5.0",
     "react-router-dom": "4.3.1",
     "react-router-redux": "5.0.0-alpha.8",
```

Comment:
react-router-redux shoould be removed from deps

Reply:
sry, somehow I missed it